mock/api.js

```diff
@@ -6,41 +6,24 @@ const DEFAULT_SIZE = 100;
 export const mockIndices = new Array(DEFAULT_SIZE).fill().map(() => moment().format('YYYY-MM'));
 
 export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-  controller: casual.word + index,
-  key: casual.word + index,
-  last_modified_string: moment.utc() + index,
+  controller: `${casual.word}.${casual.word}.com`,
+  key: casual.ip,
+  last_modified_string: moment().valueOf(moment.utc()),
   last_modified_value: moment.utc() + index,
   results: casual.integer(1, DEFAULT_SIZE),
 }));
 
-export const mockResults = {
-  took: DEFAULT_SIZE,
-  hits: {
-    total: {
-      value: DEFAULT_SIZE,
-    },
-    hits: new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-      _index: moment().format('YYYY-MM'),
-      _type: casual.word,
-      _id: casual.uuid,
-      _score: null,
-      _source: {
-        '@metadata': {
-          controller_dir: casual.word,
-        },
-        run: {
-          controller: casual.word + index,
-          name: casual.word,
-          start: moment.utc() + index,
-          end: moment.utc() + index,
-          id: casual.uuid,
-          config: casual.word,
-        },
-      },
-      sort: [casual.unix_time],
-    })),
-  },
-};
+export const mockResults = hostname =>
+  new Array(DEFAULT_SIZE).fill().map(() => ({
+    '@metadata.controller_dir': hostname,
+    config: casual.word,
+    controller: hostname,
+    end: moment.utc(),
+    // Since dataset id is a long hex string, removed "-" characters here to make it look like real data
+    id: casual.uuid.replaceAll('-', ''),
```

Comment:
Seeing the following error in development environment:

```
id: _casual.default.uuid.replaceAll('-', ''),
                           ^

TypeError: _casual.default.uuid.replaceAll is not a function
```

Reply:
Older browser version? The `replaceAll` is apparently new. Instead, `casual.uuid.replace(/-/g, '');` ... or use something other than `uuid` here, since what we want is really a "long hex string" (pretending to be an MD5 hash of the document).

Reply:
replaced with `casual.uuid.replace(/-/g, '');`